File: poo2/projeto/php/modelo/endereco.php
<?php 
    class Endereco{
        public $logradouro;
        public $cep;
        public $numero;
        public $complemento;
        public $referencia;

        function Endereco($logradouro,$cep,$numero,$complemento,$referencia){
            $this->logradouro   =$logradouro;
            $this->cep          =$cep;
            $this->numero       =$numero;
            $this->complemento  =$complemento;
            $this->referencia   =$referencia;
        }
    }
?>

File: projeto/ServeletAutopecas/src/java/LN/Endereco.java

package LN;
public class Endereco {
    private String logradouro;
    private int cep;
    private String numero;
    private String complemento;
    private String referencia;

    public String getLogradouro() {
        return logradouro;
    }

    public void setLogradouro(String logradouro) {
        this.logradouro = logradouro;
    }

    public int getCep() {
        return cep;
    }

    public void setCep(int cep) {
        this.cep = cep;
    }

    public String getNumero() {
        return numero;
    }

    public void setNumero(String numero) {
        this.numero = numero;
    }

    public String getComplemento() {
        return complemento;
    }

    public void setComplemento(String complemento) {
        this.complemento = complemento;
    }

    public String getReferencia() {
        return referencia;
    }

    public void setReferencia(String referencia) {
        this.referencia = referencia;
    }

    public Endereco(String logradouro, int cep, String numero, String complemento, String referencia) {
        this.logradouro = logradouro;
        this.cep = cep;
        this.numero = numero;
        this.complemento = complemento;
        this.referencia = referencia;
    }
    
}
